jstp/src/main/java/com/metarhia/jstp/connection/Connection.java

```diff
@@ -623,10 +736,22 @@ private boolean isSecondArray(JSObject message) {
     return message.getByIndex(1) instanceof List;
   }
 
+  /**
+   * Sets client method names for incoming inspect packets
```

Comment:
It's better to just put a link here and state that this is just a wrapper method for the one below.